### PR TITLE
Add openpubkey blog post.

### PR DESCRIPTION
**Pull request by dlorenc**

### CODE REVIEW

**Comment by dlorenc** [comment]
Thanks for the thorough reviews @kommendorkapten and @haydentherapper

**Review by kommendorkapten — APPROVED** [review]

**Comment by lukehinds** [comment]
Looks good to me thanks Dan. The only suggestion I would make is to do something like we did for the spicy blockchain / sigstore post, with an author / community disclaimer: https://blog.sigstore.dev/sigstore-blockchain-vs-transparency-logs-d673ea41a9be/ 

This is useful for if someone in the community disagrees / feels misrepresented, we can state its the authors views and not necessarily the whole community.

**Review by lukehinds — APPROVED** [review]

**Comment by lukehinds** [comment]
> > Looks good to me thanks Dan. The only suggestion I would make is to do something like we did for the spicy blockchain / sigstore post, with an author / community disclaimer: https://blog.sigstore.dev/sigstore-blockchain-vs-transparency-logs-d673ea41a9be/
> 
> Thanks, added!

Cool, TSC are aware and should review shortly, I will keep a window open for them, but realise we want to respond while the topic is fresh.

File: content/openpubkey-and-sigstore.md
Comment (imjasonh) [inline content/openpubkey-and-sigstore.md:169]:
```suggestion
The current OpenPubKey reference implementation [contains a `TODO` with just that](https://github.com/openpubkey/openpubkey/blob/11849ba7f67611180997db3676905dc26da13b20/parties/githubclient.go#L172) - fetch the keys from some log somewhere.
```

**Review by cpanato — DISMISSED** [review]

**Review by trevrosen — APPROVED** [review]

**Review by priyawadhwa — DISMISSED** [review]

**Review by bobcallaway — APPROVED** [review]

File: content/openpubkey-and-sigstore.md
Comment (SantiagoTorres) [inline content/openpubkey-and-sigstore.md:90]: I'd rephrase this as "this effectively removes the OIDC part of Fulcio (but not its transparency log)" or so

File: content/openpubkey-and-sigstore.md
Comment (SantiagoTorres) [inline content/openpubkey-and-sigstore.md:209]: Can we add "and academic partners like Purdue University" here :) ?

**Review by SantiagoTorres — APPROVED** [review]
I'm throwing it two nits. I like the way it's presented. I believe that this perhaps is not suitable for long lived things (i.e., supply chain), but perhaps it was never meant to?

**Review by priyawadhwa — APPROVED** [review]

**Comment by priyawadhwa** [comment]
cc @trevrosen @bobcallaway @SantiagoTorres @lukehinds I think we need 2 more approvals for this to merge

**Review by trevrosen — APPROVED** [review]

**Comment by joshuagl** [comment]
> I'm throwing it two nits. I like the way it's presented. I believe that this perhaps is not suitable for long lived things (i.e., supply chain), but perhaps it was never meant to? 

There's an in-toto attestation signer implementation in their GH org, which suggests they are targeting supply chain use-casea

**Review by SantiagoTorres — APPROVED** [review]

**Pull request merged**
